CMakeLists.txt

```diff
@@ -166,6 +167,7 @@ set(SRCS
   src/peerconn.c
   src/play.c
   src/reg.c
+  src/receiver.c
```

Comment:
what is "received" ?

perhaps "rtprecv.c" is a better name ?

Reply:
This comes from the `struct receiver` like currently in `main`. Should I rename only the file `receiver.c` to `rtprecv.c` and keep the simple name of the `struct receiver`?